src/sentry/static/sentry/app/views/stream/stream.jsx

```diff
@@ -97,10 +97,9 @@ const Stream = createReactClass({
 
     this.fetchSavedSearches();
     this.fetchProcessingIssues();
-  },
-
-  componentDidMount() {
-    this.fetchData();
+    if (!this.state.loading) {
```

Comment:
What are the cases when this happens? Do we have tests that cover this?

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/stream/stream.jsx#L72